openpype/hosts/blender/scripts/build_workfile.py

```diff
@@ -594,12 +596,18 @@ def build_anim(project_name, asset_name):
         )
 
         # Switch container to versioned
-        loader = get_loader(
-            project_name,
-            version_representation,
-            container_metadata.get("loader"),
-        )
-        switch_container(container_metadata, version_representation, loader)
+        loader_name = container_metadata.get("loader")
+        if not loader_name or not isinstance(loader_name, str):
+            continue
+        loader = get_loader(project_name, version_representation, loader_name)
+
+        if loader:
+            try:
+                switch_container(
+                    container_metadata, version_representation, loader
+                )
+            except RuntimeError:
+                continue
```

Comment:
I think there should at least be a warning in that case. So it still doesn't block but we have a trace that switch container fails.